src/hidpp20.c

```diff
@@ -1667,8 +1663,8 @@ union hidpp20_internal_profile {
 			char txt[16 * 3];
 			uint8_t raw[16 * 3];
 		} name;
-		uint8_t logo_effect[11]; /* G303 only */
-		uint8_t side_effects[11]; /* G303 only */
+		struct hidpp20_internal_led logo_led; /* G303, g502, g900 only */
+		struct hidpp20_internal_led side_led; /* G303, g502, g900 only */
```

Comment:
these are currently unused, so if you split out the structs into a separate patch we could merge this directly without depending on the rest of the implementation. i.e. make a patch that adds the hidpp20_internal_led and the hidpp20 color and replace it here in the struct.